gatling-core/src/main/scala/io/gatling/core/assertion/AssertionBuilders.scala

```diff
@@ -66,7 +66,13 @@ class AssertionWithPathAndTarget[T: Numeric](path: AssertionPath, target: Target
   @SuppressWarnings(Array("org.wartremover.warts.DefaultArguments"))
   def around(mean: T, plusOrMinus: T, inclusive: Boolean = true): Assertion =
     between(numeric.minus(mean, plusOrMinus), numeric.plus(mean, plusOrMinus), inclusive)
+  @SuppressWarnings(Array("org.wartremover.warts.DefaultArguments"))
+  def deviatesAround(target: T, percentDeviationThreshold: T, inclusive: Boolean = true): Assertion = {
+    val margin = numeric.fromInt((numeric.toDouble(target) * numeric.toDouble(percentDeviationThreshold) / 100).toInt)
```

Comment:
Should we truncate or round?

Reply:
its a discussionable question, we can add another paramter, but i think it will make method overweight with parameters. changing general type to Int in future releases should remov this problem, but for now i prefer to use strickter borders with truncating